### PR TITLE
Add Kiosk Mode capabilities and update syntax for macOS 10.12 in 'deprecationMsg'

### PR DESCRIPTION
**Pull request by jbaker10**

### CODE REVIEW

**Comment by jbaker10** [comment]
CLA signed.

File: deprecation_notifier/DeprecationNotifier/DNAppDelegate.m [hunk 1]
```diff
@@ -175,6 +176,16 @@ - (void)hideCountdownWindowReopen:(BOOL)reopen {
...
 
 - (void)openCountdownWindow {
+  // Start Kiosk Mode to disallow Expose and other such features
+  @try {
+      int *kioskOptions = [NSLocalizedString(@"kioskModeSettings", @"") intValue];
+      NSApplicationPresentationOptions options = kioskOptions;
+      [NSApp setPresentationOptions:options];
+  }
+  @catch(NSException * exception) {
+      NSLog(@"Error.  Invalid combination of Kiosk Mode options.");
```
Comment (russellhancox) [inline deprecation_notifier/DeprecationNotifier/DNAppDelegate.m]: Nit: s/Error.  /Error: /

File: deprecation_notifier/DeprecationNotifier/DNAppDelegate.m [hunk 2]
```diff
@@ -175,6 +176,16 @@ - (void)hideCountdownWindowReopen:(BOOL)reopen {
 }
 
 - (void)openCountdownWindow {
+  // Start Kiosk Mode to disallow Expose and other such features
+  @try {
+      int *kioskOptions = [NSLocalizedString(@"kioskModeSettings", @"") intValue];
+      NSApplicationPresentationOptions options = kioskOptions;
+      [NSApp setPresentationOptions:options];
+  }
+  @catch(NSException * exception) {
```
Comment (russellhancox) [inline deprecation_notifier/DeprecationNotifier/DNAppDelegate.m]: Combine closing brace and @exception and join up *exception:

```objc
} @catch (NSException *exception) {
```

**Comment by russellhancox** [comment]
Thanks for the PR!

**Comment by jbaker10** [comment]
Thank you!

**Pull request merged**